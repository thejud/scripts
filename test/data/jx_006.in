{"foo": 1, "bar": "0134", "baz": "1234567890"}
{"foo": 12, "bar": "a", "baz": "00"}



