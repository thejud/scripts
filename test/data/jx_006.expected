foo  bar   baz
1    0134  1234567890
12   a     00
